content/en/blog/_posts/2024-09-20-kubeadm-v1beta4.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog
+title: 'Kubernetes v.31: kubeadm v1beta4'
+date: 2024-09-20
+slug: kubernetes-1-31-kubeadm-v1beta4
+author: >
+   Paco Xu (DaoCloud),
+---
+
+[`kubeadm`](/docs/reference/setup-tools/kubeadm/) Configuration([v1beta4](https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta4/)) is released with Kubernetes v1.31.
+The configuration 
+v1beta3 is still supported, but v1beta3 will be deprecated and removed in future releases.
```

Comment:
```suggestion
The [`kubeadm`](/docs/reference/setup-tools/kubeadm/) configuration
([v1beta4](https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta4/))
is released with Kubernetes v1.31. While v1beta3 is still supported,
it will be deprecated and removed in future releases.
```